src/App.svelte

```diff
@@ -113,23 +98,18 @@
           addressOrName={$activeRouteStore.params.addressOrName}
           {wallet} />
       {:else if $activeRouteStore.resource === "404"}
-        <NotFound title="404" subtitle="Nothing here" />
+        <div class="wrapper" style="justify-content: center;">
+          <NotFound title="404" subtitle="Nothing here" />
+        </div>
       {:else}
         {unreachable($activeRouteStore)}
       {/if}
     </div>
   {:catch err}
-    <div class="wrapper">
-      <Modal error subtle>
-        <span slot="title">
-          <div class="emoji" use:twemoji>👻</div>
-          <div>Error connecting to network</div>
-        </span>
-
-        <span slot="body">
-          {err.message ? err.message : JSON.stringify(err)}
-        </span>
-      </Modal>
+    <div class="wrapper" style="justify-content: center;">
```

Comment:
The same as on L101

Reply:
b556f3b9587b96fa099dbadf7067cce88424c9f5